modepy/matrices.py

```diff
@@ -66,7 +72,10 @@
 """
 
 
-def vandermonde(functions, nodes):
+def vandermonde(
+            functions: Sequence[Callable[[np.ndarray], np.ndarray]],
+            nodes: np.ndarray
+        ) -> np.ndarray:
```

Comment:
Couldn't add a comment below, but this could delete the sentence in the docs below with `*functions* are allowed to return...`?

Reply:
Done.